src/App.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import logo from './logo.svg';
 import { Counter } from './features/counter/Counter';
 import './App.css';
+import FieldInput from './features/input/FieldInput';
```

Comment:
Counter and FieldInput are very similar, but are imported differently, they should both be imorted as `import { C } from` or `import C from`.